src/crab/array_domain.cpp

```
@@ -4,201 +4,74 @@
+        return offset < o + sz && o < offset + size;
     }
-
-    // Return true if [symb_lb, symb_ub] may overlap with the cell,
-    // where symb_lb and symb_ub are not constant expressions.
-    [[nodiscard]]
-    bool symbolic_overlap(const Interval& range) const;
-
-    friend std::ostream& operator<<(std::ostream& o, const Cell& c) { return o << "cell(" << c.to_interval() << ")"; }
 };
 
+static Interval cell_to_interval(const offset_t o, const unsigned size) {
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)